docker_explorer/de.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python
```

Comment:
Perhaps /usr/bin/env python so that this works in virtualenvs as well?

Reply:
I opt to make this part of the python3 changes, seeing that now only `#!/usr/bin/python2` really works

Reply:
Updated #7